Otherwise using VPU doesn't activate SPLL (if not already up with USB
for example) and VPU firmware loading crashes at first coda_write().

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: linux-4.4/drivers/clk/imx/clk-imx27.c
===================================================================
--- linux-4.4.orig/drivers/clk/imx/clk-imx27.c
+++ linux-4.4/drivers/clk/imx/clk-imx27.c
@@ -29,7 +29,7 @@ static void __iomem *ccm __initdata;
 #define CCM_PCCR1		(ccm + 0x24)
 #define CCM_CCSR		(ccm + 0x28)
 
-static const char *vpu_sel_clks[] = { "spll", "mpll_main2", };
+static const char *vpu_sel_clks[] = { "spll_gate", "mpll_main2", };
 static const char *cpu_sel_clks[] = { "mpll_main2", "mpll", };
 static const char *mpll_sel_clks[] = { "fpm", "mpll_osc_sel", };
 static const char *mpll_osc_sel_clks[] = { "ckih_gate", "ckih_div1p5", };
